lib/packages/core/src/burstUtil.ts

```diff
@@ -0,0 +1,276 @@
+import { HttpError } from "@burst/http";
+import { HttpResponse } from "@burst/http";
+import * as BN from "bn.js";
+
+/*
+* Copyright 2018 PoC-Consortium
+*/
+
+/*
+* BurstUtil class
+*
+* The BurstUtil class provides static methods for encoding and decoding numeric ids.
+* In addition, addresses can be checked for validity.
+*/
+export class BurstUtil {
```

Comment:
Todo: Break this class out into individual functions (see `core/getAccountIdFromBurstAddress.ts`)

Reply:
Actually might want to move this into its own package, i.e. `@burst/utils`